build/wrap-tables.ts

```diff
@@ -4,6 +4,8 @@
 import * as cheerio from "cheerio";
 
 export function wrapTables($: cheerio.CheerioAPI) {
-  const figure = $('<figure class="table-container"></figure>');
+  const figure = $(
+    '<figure class="table-container"><figure class="table-container-inner"></figure>'
+  );
```

Comment:
I think the solution might actually be to add a CSS class for these tables that DON'T have the nested `table-container-inner`:

```suggestion
  const figure = $(
    '<figure class="table-container"><figure class="table-container-inner"></figure>'
  );
```
```suggestion
  const figure = $('<figure class="table-container standalone"></figure>');
```